build/Dockerfile

```diff
@@ -5,9 +5,9 @@ ENV OPERATOR=/home/kubedirector/kubedirector \
     USER_NAME=kubedirector \
     USER_HOME=/home/kubedirector
 
-# Support ps and killall, for redeploy.
+# Support ps, killall, and tar for redeploy.
 RUN microdnf update -y && rm -rf /var/cache/yum
-RUN microdnf -y install --nodocs psmisc procps-ng \
+RUN microdnf -y install --nodocs psmisc procps-ng tar \
```

Comment:
This is actually just something I didn't notice about the recent change to using UBI 8. Apparently tar is no longer included by default, and we need it to support "kubectl cp".